<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CSimuladorPoco</class>
 <widget class="QMainWindow" name="CSimuladorPoco">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1052</width>
    <height>402</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CSimuladorPoco</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>371</width>
      <height>311</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="4" column="0">
      <widget class="QLabel" name="lbnTituloDiametroOD">
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Diamentro Revestimento OD (in)</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="lbnTituloVazao">
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Vazão (gal/min)</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="13" column="1" colspan="2">
      <widget class="QPushButton" name="btnAtualizarDados">
       <property name="text">
        <string>Atualizar Dados</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QLineEdit" name="editNomePoco"/>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="QLineEdit" name="editProfundidadeTotal"/>
     </item>
     <item row="4" column="1" colspan="2">
      <widget class="QLineEdit" name="editDiametroOD"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="lbnTituloNomePoco">
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Nome Poço</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lbnTituloProfunidadeTotal">
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Profundidade Total (ft)</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="5" column="1" colspan="2">
      <widget class="QLineEdit" name="editDiametroID"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="lbnTituloPressaoSuperficie">
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Pressão Superficie (psi)</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="12" column="0" colspan="3">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="3" column="1" colspan="2">
      <widget class="QLineEdit" name="editDiametroPoco"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="lbnTituloDiametroPoco">
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Diametro Poço (in)</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="13" column="0">
      <widget class="QPushButton" name="btnAdicionarPropriedades">
       <property name="text">
        <string>Adicionar Propriedade</string>
       </property>
      </widget>
     </item>
     <item row="10" column="1" colspan="2">
      <widget class="QLabel" name="lbnDensidadeMedia">
       <property name="text">
        <string>-</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="lbnTituloProfundidadeOcupada">
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Produndidade Ocupada (ft): </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="11" column="1" colspan="2">
      <widget class="QLabel" name="lbnViscosidadeMedia">
       <property name="text">
        <string>-</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="9" column="1" colspan="2">
      <widget class="QLabel" name="lbnProfundidadeOcupada">
       <property name="text">
        <string>-</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="2">
      <widget class="QLineEdit" name="editPressaoSuperficie"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="lbnTituloDiametroID">
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Diamentro Revestimento ID  (in)</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="6" column="1" colspan="2">
      <widget class="QLineEdit" name="editVazao"/>
     </item>
     <item row="11" column="0">
      <widget class="QLabel" name="lbnTituloViscosidadeMedia">
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Viscosidade Média (cP):</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="lbnTituloDensidadeMedia">
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Densidade Média (lb/gal): </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="8" column="0" colspan="3">
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>30</y>
      <width>601</width>
      <height>311</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="1" column="0">
      <widget class="QPushButton" name="btnAdicionarFluido">
       <property name="text">
        <string>Adicionar Fluido</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="btnRemoverFluido">
       <property name="text">
        <string>Remover Fluido</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="0" colspan="3">
      <widget class="QTableWidget" name="tblFluidos">
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>120</number>
       </attribute>
       <column>
        <property name="text">
         <string>Nome</string>
        </property>
        <property name="font">
         <font>
          <bold>true</bold>
         </font>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Densidade (lbm/gal)</string>
        </property>
        <property name="font">
         <font>
          <bold>true</bold>
         </font>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Viscosidade (cP)</string>
        </property>
        <property name="font">
         <font>
          <bold>true</bold>
         </font>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Prof. Inicial (ft)</string>
        </property>
        <property name="font">
         <font>
          <bold>true</bold>
         </font>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Prof. Final (ft)</string>
        </property>
        <property name="font">
         <font>
          <bold>true</bold>
         </font>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1052</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArquivo">
    <property name="title">
     <string>Arquivo</string>
    </property>
    <addaction name="actionImportar_Dados"/>
   </widget>
   <addaction name="menuArquivo"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionImportar_Dados">
   <property name="text">
    <string>Importar Dados</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
